theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
As discussed on slack, should we refactor `@sk-yellow` to be new colour documented in the guidelines, and assign the old value to `@sk-premium-job-highlight`?